font/font.go

```diff
@@ -0,0 +1,12 @@
+package font
+
+import (
+	"github.com/benoitkugler/textlayout/fonts"
+	"github.com/benoitkugler/textlayout/fonts/truetype"
+)
+
+type Resource = fonts.Resource
+
+func ParseTTF(file Resource, loadAllTables bool) (Face, error){
```

Comment:
Actually, in the context of this package, we always want to load all the tables.
So I would remove `loadAllTables` from the `ParseTTF` arguments (and pass `true` to the  `truetype.Parse` call.